color/preview.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @file
+ * This is simple js for management of color module display.
+ */
+
+(function ($, Drupal, drupalSettings) {
```

Comment:
seems drupalSettings not needed here. same as jQuery imho.

Reply:
Fixed.